README.md

```diff
@@ -7,9 +7,10 @@
 [![](https://images.microbadger.com/badges/image/gitea/gitea.svg)](http://microbadger.com/images/gitea/gitea "Get your own image badge on microbadger.com")
 [![Join the chat at https://gitter.im/go-gitea/gitea](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/go-gitea/gitea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![](https://github.com/go-gitea/gitea/blob/master/public/img/gogs-large-resize.png?raw=true)
+![](https://github.com/go-gitea/gitea/blob/master/public/img/gitea-large-resize.png?raw=true)
 
-##### Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) for binary versions or submit a task on [alpha stage automated binary building system](https://build.gogs.io/))
+##### Status
+## Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) 
```

Comment:
agree @lunny 
i though at first to remove this line and simply link to the release page.